release_notes/365-release-notes.adoc

```diff
@@ -0,0 +1,87 @@
+[id="365-release-notes"]
+= Red Hat Advanced Cluster Security for Kubernetes 3.65
+include::modules/common-attributes.adoc[]
+:context: 365-release-notes
+
+toc::[]
+
+{product-title} (RHACS) 3.65 includes feature enhancements, bug fixes, scale improvements, and other changes.
+
+*Release date:* September 6, 2021
+
+[id="new-features"]
+== New features
+
+[id="mitre-attck-framework"]
+=== MITRE ATT&CK framework
+You can now use the link:https://attack.mitre.org/matrices/enterprise/containers/[MITRE ATT&CK Framework] to categorize policies in the {product-title}. When you xref:../operating/manage-security-policies.adoc#create-custom-policies[create custom security policies], you can now add MITRE ATT&CK framework's adversary tactics and techniques related information.
```

Comment:
nit: The MITRE thing seems treated like a proper noun.  My sugg is to either repeat with the upcase F in the second instance of Framework, or replace s/MITRE ATT&CK framework's/framework's/ and refer to it simply as a framework. Up to you.

Reply:
I navigated the page and changed the term to `Matrix` based on the description at https://attack.mitre.org/matrices/enterprise/containers/